external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -88,7 +88,10 @@ private[kafka010] case class KafkaSource(
 
   private val sc = sqlContext.sparkContext
 
-  private val pollTimeoutMs = sourceOptions.getOrElse("kafkaConsumer.pollTimeoutMs", "512").toLong
+  private val pollTimeoutMs = sourceOptions.getOrElse(
+    "kafkaConsumer.pollTimeoutMs",
+    (sc.conf.getTimeAsSeconds("spark.network.timeout", "120s") * 1000).toString
```

Comment:
Sorry, I think I was unclear and it is really minor.  I agree we should keep it as `120s` for consistency.  I was only suggesting we use the built conversion done by `getTimeAsMs` rather than embedding the magic `* 1000` here.

```scala
sc.conf.getTimeAsMs("spark.network.timeout", "120s")
```